### PR TITLE
[nan-748] nested script structure update

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Update docs to reflect nested structure update

Adds this snippet
image

## Issue ticket number and link
NAN-748

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/customize/guides/create-a-custom-integration.mdx [hunk 1]
```diff
@@ -91,7 +91,7 @@ This command starts a process that continuously compiles your integration script
 Open the generated sync script (named `[sync-name].ts`) which should contain the following scaffolding : 
 
 ```typescript salesforce-contacts.ts
-import { NangoSync, Contact } from './models';
+import { NangoSync, Contact } from '../../models';
```
Comment (bodinsamuel) [inline docs-v2/customize/guides/create-a-custom-integration.mdx]:
```suggestion
import type { NangoSync, Contact } from '../../models';
```

File: docs-v2/customize/guides/create-a-custom-integration.mdx [hunk 2]
```diff
@@ -101,7 +101,7 @@ export default async function fetchData(nango: NangoSync): Promise<void> {
 Fill in the `fetchData` method with your integration code (in the example here, we fetch tasks from Salesforce):
 
 ```ts salesforce-contacts.ts
-import { NangoSync, Contact } from './models';
+import { NangoSync, Contact } from '../../models';
```
Comment (bodinsamuel) [inline docs-v2/customize/guides/create-a-custom-integration.mdx]:
```suggestion
import type { NangoSync, Contact } from '../../models';
```

**Review by bodinsamuel — APPROVED** [review]

**Review by TBonnin — APPROVED** [review]

File: docs-v2/reference/cli.mdx [hunk 3]
```diff
@@ -42,13 +54,14 @@ Get details about a specific command by running:
...
+| `nango dev`                    | Necessary to edit integration configurations and scripts.                                                                                         | Watches the `nango.yaml` and integration scripts, re-generates `models.ts` and compiled `.js` files. |
+| `nango generate`               | Generates an integration script `.ts` file with initial scaffold when new syncs appear in your `nango.yaml`.                                      | Re-generates `models.ts`, not the compiled `.js` files.                                              |
+| `nango dryrun <params>`        | Lets you test integration scripts locally.                                                                                                        |                                                                                                      |
+| `nango deploy <env>`           | Lets you deploy your sync to your `dev` or `prod environment`                                                                                     | Generates the compiled `.js` files.                                                                  |
```
Comment (TBonnin) [inline docs-v2/reference/cli.mdx]: nitpicking: `dev` or `prod` environment

**Review by TBonnin — APPROVED** [review]

File: docs-v2/reference/cli.mdx [hunk 4]
```diff
@@ -13,7 +13,19 @@ npm install nango -g
...
+```
+
+This creates the `./nango-integrations` folder with some initial configuration and an example
+sync script. The `nango-integrations` directory looks like this:
+```
+nango-integrations/
+├── .env
+├── nango.yaml
+├── models.ts
+└── demo-github-integration
```
Comment (bastienbeurier) [inline docs-v2/reference/cli.mdx]: I would indicate with a comment that this is the integration ID, and must match an entry in the `nango.yaml` 
 and UI.

File: docs-v2/reference/cli.mdx [hunk 5]
```diff
@@ -13,7 +13,19 @@ npm install nango -g
...
+    └── syncs/
+        └── github-issue-example.ts
```
Comment (bastienbeurier) [inline docs-v2/reference/cli.mdx:28]: I would indicate with a comment that this is is matching the script name in the `nango.yaml`.

Comment (khaliqgant) [reply docs-v2/reference/cli.mdx:28]: Done, thanks.

File: docs-v2/reference/cli.mdx (hunk 3 above)
Comment (khaliqgant) [reply docs-v2/reference/cli.mdx]: Updated.

**Pull request merged**